web3/eth/async_eth.py

```diff
@@ -518,6 +521,31 @@ async def get_storage_at(
     ) -> HexBytes:
         return await self._get_storage_at(account, position, block_identifier)
 
+    # eth_sign
+
+    def sign_munger(
```

Comment:
This method can get moved to BaseEth and then called on line 537 as: `BaseEth.sign_munger`